magento/magento-cloud
# Hotifx for fatal error during deployment for 2.3.5-p1

### Description
Due to wrong `.magento.app.yaml` configuration deployment on Magento Cloud is not possible on 2.3.5-p1

### Manual testing scenarios
1. Init Magetno Cloud repo with this repo files
2. Execute deployment
3. The following error will appear:
```
[2020-06-15 11:12:55] INFO: Starting scenario(s): scenario/post-deploy.xml (magento/ece-tools version: 2002.1.0, magento/magento2-base version: 2.3.5-p1)
[2020-06-15 11:12:55] NOTICE: Validating configuration
[2020-06-15 11:12:56] NOTICE: End of validation
[2020-06-15 11:12:56] INFO: Enable cron
[2020-06-15 11:12:56] INFO: Create backup of important files.
[2020-06-15 11:12:56] INFO: Backup /app/etc/env.php.bak for /app/etc/env.php was created.
[2020-06-15 11:12:56] INFO: Backup /app/etc/config.php.bak for /app/etc/config.php was created.
[2020-06-15 11:12:57] ERROR: The command "php ./bin/magento cache:flush --ansi --no-interaction" failed. In ErrorHandler.php line 61:
```
```
Warning: array_replace_recursive(): recursion detected in vendor/magento/module-config/App/Config/Type/System.php on line 195
```

Internal Magento Support ticket: **321325**
